src/test_workflow/test_args.py

```diff
@@ -36,9 +38,21 @@ def __init__(self):
         self.component = args.component
         self.keep = args.keep
         self.logging_level = args.logging_level
-        self.path = args.path if validators.url(args.path) else os.path.realpath(args.path)
+        self.test_manifest_path = self._validate_path(args.test_manifest_path)
+
+        self.opensearch_path = self._validate_paths(args.paths, "opensearch")
+        self.opensearch_dashboards_path = self._validate_paths(args.paths, "opensearch-dashboards")
```

Comment:
When running OpenSearch distribution tests the *dashboards_path tests is not needed, could you abstract this validation so its based on a test manifest settings?

Reply:
This will require loading manifest content. Seems a bit heavy weight for args parsing stage. We can have a follow up item on this if you prefer such checks.

Reply:
Please create a follow up issue

Reply:
https://github.com/opensearch-project/opensearch-build/issues/1276